pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-7713.xml

```diff
@@ -4,7 +4,7 @@
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
                    https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
 
-    <changeSet author="Dmitrii Eliseev" id="2022-05-12-tabular-block">
+    <changeSet author="Dmitrii Eliseev" id="2022-06-08-tabular-block">
```

Comment:
I assume you need to create a new changeset. It was merged to dev and applied, so no changes could be done here